packages/api/README.md

```diff
@@ -56,3 +56,38 @@ stripe trigger payment_intent.succeeded
 ```
 
 To test payments, you can use the [Stripe test cards](https://stripe.com/docs/testing#cards).
+
+## Local development with Azure resources
+
+To develop the Azure Functions app with the Azure databases, use the following steps.
+
+1. Fork the repository and open in Codespaces or Visual Studio dev container.
```

Comment:
note: while running the repo in Codespaces or devcontainer is technically the same thing, we only support Codespaces.

Reply:
@manekinekko How would that be seen in the project to a developer new to the project?